docs/index.rst

```diff
@@ -1,17 +0,0 @@
-ScippUncertainty
-================
-
-Advanced uncertainty propagation with Scipp.
-
-`Scipp <https://scipp.github.io/>`_ provides a basic facility for storing and propagating variances.
-It can, however, not take correlations into account.
-ScippUncertainty provides tools to this end.
-Currently, only Monte-Carlo uncertainty propagation is implemented.
-See the corresponding `user guide <./user-guide/mc.rst>`_.
-
-.. toctree::
-   :hidden:
-
-   user-guide/index
-   reference/index
-   references
```

Comment:
These two show up as `Reference` and `References` in the docs:
![Screenshot at 2023-03-17 11-12-35](https://user-images.githubusercontent.com/39047984/225876310-486f0cbe-0f4c-46f8-b211-035052215787.png)

Can we rename the second one to `Bibliography` to make it less confusing?